graphql/factory/plan.go

```diff
@@ -52,10 +53,11 @@ func PlanFromDomainModel(plan models.Plan, startLocation *models.GeoLocation) (*
 	}
 
 	return &graphql.Plan{
-		ID:            plan.Id,
-		Name:          plan.Name,
-		Places:        places,
-		TimeInMinutes: int(plan.TimeInMinutes),
+		ID:     plan.Id,
+		Name:   plan.Name,
+		Places: places,
+		// MOCK: transitionを利用し時間を計算する関数を実装する
```

Comment:
@0re2236 
エディタでTODOコメントを検知してくれることが多いので、TODOとしたほうが良さそうです
```suggestion
		// TODO: transitionを利用し時間を計算する関数を実装する
```

また、このモデルを利用すると、時間が取得できそうです！
https://github.com/poroto-app/planner/blob/develop/internal/domain/models/transition.go